packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx

```diff
@@ -46,7 +46,7 @@ export const CreateOrganizationForm = (props: CreateOrganizationFormProps) => {
   });
 
   const dataChanged = !!nameField.value;
-  const canSubmit = dataChanged || !!file;
+  const canSubmit = dataChanged;
```

Comment:
Does the avatar get uploaded without submitting the form? Is that why we don't check for it here?

Reply:
Also, is there not `form.isDirty` or something similar in clerk-js?

Reply:
We upload the logo, after the org is created (after the submission)

No i don't think we support isDirty out of the box.